pkg/apis/openebs.io/zfs/v1alpha1/zfsvolume.go

```diff
@@ -192,6 +192,11 @@ type VolumeInfo struct {
 	// FsType can not be modified once volume has been provisioned.
 	// Default Value: ext4.
 	FsType string `json:"fsType,omitempty"`
+
+	// ShareNfs specifies if, and when, how, to share the dataset via NFS.
+	// It depends on the Linux NFS Kernel Server to be running on the machine
+	// providing the volume and the volume to be mounted.
+	ShareNfs string `json:"sharenfs,omitempty"`
```

Comment:
we have shifted to v1 apis, can remove this from here and make the changes in pkg/apis/openebs.io/zfs/v1 directory where v1 apis are there?

Reply:
Sure ... I will touch it again later this week. Currently I am reaching out to openzfs, because I do not understand how the `zfs share` code is supposed to interact with the `nfs-kernel-server`. 

Calling `execsnoop`, I see `zfs share` forking off to `exportfs`, but even with `hostIPC` and `hostPID`, I do not see the share being exported. Successful NFS export relies on calling `zfs share` from the host, which clearly is undesirable in our case.

You wouldn't happen to have an idea what might be happing there?

Reply:
let me check that.